INTERIOR/GetInteriorHeading.md

```diff
@@ -1,18 +1,15 @@
 ---
 ns: INTERIOR
 ---
-## _GET_INTERIOR_HEADING
+## GET_INTERIOR_HEADING
 
 ```c
 // 0xF49B58631D9E22D9
-float _GET_INTERIOR_HEADING(int interior);
-```
-
-```
-GET_INTERIOR_*
-
-NativeDB Introduced: v1493
+float GET_INTERIOR_HEADING(int interior);
 ```
 
 ## Parameters
-* **interior**:
+* **interior**: ID of the interior
+
+## Return value
+Returns interior heading in radians. Multiply the returned value with 57.29578 to get it in degrees.
```

Comment:
Just a quick note, that `57.29578` is the result of [180/Pi](https://en.wikipedia.org/wiki/Pi) or [360/τ](https://en.wikipedia.org/wiki/Turn_(angle)#Tau_proposals) for people that prefer such.

### Pi = π (3.14159...)
### Tau = 2π (6.28318...)

Reply:
I changed the return value text, how is it now?

Reply:
> I changed the return value text, how is it now?

LGTM now, thank you 👍